projects/demo-integrations/cypress/tests/kit/slider/slider.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    DEFAULT_TIMEOUT_BEFORE_ACTION,
+    SLIDER_PAGE_URL,
+} from '../../../support/shared.entities';
+
+const goToApiPage = () => {
+    cy.get('[tuitab]').eq(1).click();
+};
```

Comment:
can we move this to shared cypress utilities?

Reply:
i am not sure that `eq(1)` will work for all demo-pages((

Also, use tab name "API" to build selector is a bad idea because it will not work in proprietary demo-app (it has russian tab names and url)